compiler/lib/src/main/scala/codegen/ComputeGeneratedFiles.scala

```diff
@@ -17,10 +17,10 @@ object ComputeGeneratedFiles {
 
   def getCppFiles(a: Analysis, tul: List[Ast.TransUnit]): Result.Result[List[String]] =
     for {
-      s <- ComputeCppFiles.visitList(
+      s <- ComputeAutocodeCppFiles.visitList(
         CppWriterState(a),
         tul,
-        ComputeCppFiles.transUnit
+        ComputeAutocodeCppFiles.transUnit
       )
     }
     yield s.locationMap.toList.map(_._1)
```

Comment:
This still needs some work -- I'm not sure what's the best way to indicate which filenames to compute. Should we add flags to the fpp-depend and fpp-filenames tools?

Reply:
Yes, good idea! I added draft specs for the `-t` options to the wiki. Please take a look and see what you think.

https://github.com/fprime-community/fpp/wiki/fpp-depend
https://github.com/fprime-community/fpp/wiki/fpp-filenames

Reply:
This looks good, thanks!